sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientBuilderTest.java

```diff
@@ -128,6 +130,18 @@ public void timeoutPolicy() {
         assertThrows(RuntimeException.class, () -> client.setConfigurationSetting(key, null, value));
     }
 
+    @Test
+    @DoNotRecord
```

Comment:
Is this annotation required? I guess this test only creates the client and there will be no service request made.

Reply:
Yes this is necessary, most of tests in this class are annotated with this. It makes base test class skip the recording part. This is just trying to fit into existing tests pattern here.